tcms_django_plugin/result.py

```diff
@@ -116,11 +116,22 @@ def addError(self, test, err):
         super().addError(test, err)
         self.debug_sql_stream.seek(0)
         self.errors[-1] = self.errors[-1] + (self.debug_sql_stream.read(),)
+        self.backend.add_comment(self.test_execution_id, self.errors[-1][2])
```

Comment:
This appears to be empty, see comments for TC 567, 568, 573 at https://tcms.kiwitcms.org/runs/679/ and compare them to the same executions in TR 678.

Reply:
IDK if there is nothing in the sql stream here or if the implementation is incorrect.

Reply:
I should've double-checked, I think its because there isn't any database use to get any SQL. I think I need to set up more testing in the testapp that uses the database.

Reply:
Now I see SQL statements in comments under https://tcms.kiwitcms.org/runs/681/